docs/20-package-management.md

```diff
@@ -4,7 +4,19 @@ The doAzureParallel package allows you to install packages to your pool in two w
 - Installing on pool creation
 - Installing per-*foreach* loop
 
+Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any specific dependencies required only for that instance of the loop.
+
 ## Installing Packages on Pool Creation
+
+Pool level packages support CRAN, GitHub and BioConductor packages. The packages are installed in a shared directory on the node. It is important to note that it is required to explicitly load any packages installed at the cluster level within the foreach loop. For example, if you installed xml2 on the cluster, you must explicityly load it before using it.
+
+```R
+foreach (i = 1:4) %dopar% {
+  # Load the libraries you want to use.
+  library(xml2)
```

Comment:
Is this true? I thought you can do .packages() and it'll automatically be called.

Reply:
@brnleehng - I thought that was only true for job-level packages, right? For cluster level packages I think we need to either explicitly load the library or do __xml2::__

Reply:
For the job level, we will try to reinstall and reference it with library. For the cluster level, we would need xml2::